src/acme/pd/Customer.java

```diff
@@ -85,4 +86,14 @@ public MapIntersection getIntersection() {
     public void setIntersection(MapIntersection intersection) {
         this.intersection = intersection;
     }
+
+    public static int getNextCustomerNumber() {
+		try {
+			//FIXME
```

Comment:
This is seriously broken. Can you just put a simple incrementer in there for the time being and write defect against it? it is difficult to test when this breaks each time.

Reply:
Always returns 1 now until this is fixed